### PR TITLE
[DTRA] Maryia/OPT-862/feat: icons for new Step Indices

### PR DESCRIPTION
**Pull request by maryia-deriv**
- to add icons for new Step Indices
- to update an icon for an existing Step Index

### CODE REVIEW

**Review by kate-deriv — APPROVED** [review]

File: src/components/Icons.tsx [hunk 1]
```diff
@@ -559,7 +561,9 @@ export const ItemIconMap = {
     CRASH1000: Wrapper(Crash1000),
     RDBEAR: Wrapper(MarketBear),
     RDBULL: Wrapper(MarketBull),
-    stpRNG: Wrapper(Step),
+    stpRNG: Wrapper(Step100),
+    stpRNG2: Wrapper(Step200),
```
Comment (ahmadtaimoor-deriv) [inline src/components/Icons.tsx:565]: Please ignore if irrelevant . If there is a reason the name `stpRNG` is in Uppercase in deriv-app and camelCase here?

**Review by balakrishna-deriv — APPROVED** [review]

File: src/components/Icons.tsx (hunk 1 above)
Comment (maryia-deriv) [reply src/components/Icons.tsx:565]: @ahmadtaimoor-deriv the reason is that here it's taken from `active_symbols` response while in deriv-app - from `shortcode` where it's in camel case)

Comment (ahmadtaimoor-deriv) [reply src/components/Icons.tsx:565]: Thanks

**Review by ahmadtaimoor-deriv — APPROVED** [review]

**Pull request merged**